package.json

```diff
@@ -392,6 +392,21 @@
                     "default": false,
                     "description": "Set to true to show timestamps by default in the log viewer."
                 },
+                "vscode-kubernetes.log-viewer.since": {
+                    "type": "integer",
+                    "default": -1,
+                    "description": "The default value for the since field in the log viewer. Set to -1 for all logs."
+                },
+                "vscode-kubernetes.log-viewer.tail": {
+                    "type": "integer",
+                    "default": -1,
+                    "description": "The number of recent logs to display by default in the log viewer. Set to 0 or -1 for all log lines."
+                },
+                "vscode-kubernetes.log-viewer.terminal": {
+                    "type": "boolean",
```

Comment:
Consider making this a "destination" with permitted values of `webview` and `terminal` or something like that?  (Just in case we one day come up with _yet another_ way to display logs!)

(You can still convert it to a boolean for code purposes if it's more convenient of course.)